Polygon.js

```diff
@@ -0,0 +1,129 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?://(?:www\\.)polygon\\.com",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-12-06 02:04:51"
+}
+
+// http://en.wikipedia.org/wiki/User_talk:Czar
+
+function detectWeb(doc, url) {
+	var pageType = ZU.xpathText(doc, '//script[contains(., "page_type :")]').split('page_type : \"')[1].split(/\"/)[0];
+	if (pageType == "Article") { return "webpage" } 
+	return null;
+}
+
+function scrape(doc, url) {
+	var newItem = new Zotero.Item("webpage");
+	newItem.websiteTitle = "Polygon";
+	newItem.language = "en-US";
+
+	newItem.url = doc.location.href;
+	newItem.title = ZU.xpathText(doc, '//h2[@class="m-entry__title"]');
+	newItem.date = ZU.xpathText(doc, '//meta[@property="article:published_time"]/@content').split("T")[0];
+
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	newItem.notes = [ ZU.xpathText(doc, '//*[@id="article-body"]').trim() ];
+	newItem.attachments.push({document:doc, title:"Polygon snapshot" ,mimeType:"text/html"});
+
+		// Authors
+	var authors = ZU.xpathText(doc, '//script[contains(., "dataLayer =")]').split('\"Author\":\"')[1].split(/\",/)[0].split(":");
+	newItem.creators = new Array();
+	do {
+		newItem.creators.push({ 
+			firstName:authors[0].substring(0,authors[0].lastIndexOf(" ")), 
+			lastName:authors[0].substring(authors[0].lastIndexOf(" ")+1), 
+			creatorType:"author" 
+		});
+		authors.shift();
+	}
+	while (authors.length);
+	
+	newItem.complete();
+}
+
+function doWeb(doc, url) {
```

Comment:
To get you a sense of what I mean by multiples, try something like:
```
	if (detectWeb(doc, url) == "multiple") {
	    var items = {};
	    articles = [];
	    var titles = ZU.xpath(doc, '//div[@class="m-block__meta meta"]/h2/a')
	    for (var i = 0; i < titles.length; i++) {
	        items[titles[i].href] = titles[i].textContent;
	    }
	    Zotero.selectItems(items, function(items) {
	        if (!items) {
	            return true;
	        }
	        for (var i in items) {
	            articles.push(i);
	        }
	        Zotero.Utilities.processDocuments(articles, scrape);
	    })
	}
```

(followed by `else` obviously). You'd then have to test for multiples in `detectWeb`, too, something like 
```	
else if (url.indexOf("/search?q=")!=-1){
		return "multiple"
	}
```

This would be for search results. You can see if you also want to include landing pages. Does that make sense? I realize not your core interest since I don't think Citoid ever uses this, but it makes the translator a lot more useful for researchers wanting to ingest large amounts of data.